lib/identity_cache/query_api.rb

```diff
@@ -180,7 +180,7 @@ def coder_from_record(record) #:nodoc:
           coder = {
             attributes: record.attributes_before_type_cast.dup,
           }
-          coder[:class] = record.class.name if IdentityCache::CACHE_VERSION == 7
+          coder[:class] = record.class.name
```

Comment:
I don't understand why this was under a conditional before. I'm assuming the constant won't change?

Reply:
This entry is no longer needed.  The intention was to remove it when the cache version got bumped.  I guess I should have added a comment making that clearer.